art/attacks/evasion/carlini.py

```diff
@@ -546,9 +547,11 @@ def __init__(
         :param learning_rate: The initial learning rate for the attack algorithm. Smaller values produce better
                 results but are slower to converge.
         :param max_iter: The maximum number of iterations.
-        :param max_halving: Maximum number of halving steps in the line search optimization.
-        :param max_doubling: Maximum number of doubling steps in the line search optimization.
-        :param eps: An upper bound for the L_0 norm of the adversarial perturbation.
+        :param decrease_factor: The rate of shrinking tau, values in `0 < decrease_factor < 1` where larger is more
+                                accurate.
+        :param initial_const: The initial value of constant `c`.
+        :param largest_const: The largest value of constant `c`.
+        :param const_factor: The rate of increasing constant `c` with `const_factor > 1`, where smaller more accurate.
         :param batch_size: Size of the batch on which adversarial samples are generated.
```

Comment:
If removal of `batch_size` is intended, then this line can be removed as well.

Reply:
I agree.